src/test_l1_hdmi_cec_driver.c

```diff
@@ -532,15 +528,15 @@ void test_hdmicec_hal_l1_getPhysicalAddress_positive( void )
 
     result = HdmiCecOpen (&handle);
     //if init is failed no need to proceed further
-    UT_ASSERT_EQUAL ( result, HDMI_CEC_IO_SUCCESS );
+    UT_ASSERT_EQUAL_FATAL ( result, HDMI_CEC_IO_SUCCESS );
 
     result = HdmiCecGetPhysicalAddress(handle, &physicalAddress);
-    if (HDMI_CEC_IO_SUCCESS  != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecGetPhysicalAddress failed"); }
-    UT_ASSERT_TRUE(physicalAddress==0xffff);
+    if (HDMI_CEC_IO_SUCCESS  != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecGetPhysicalAddress failed"); }
+    UT_ASSERT_TRUE_FATAL(physicalAddress!=0xffff);
```

Comment:
we need to terminate here or don't abort from here.. so that it will get terminated in the next step.

Reply:
update with new logic to avoid confuse 
if(physicalAddress == 0xffff){
            UT_FAIL("Invalid physicalAddress 0x%x",physicalAddress);
   }